release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -474,9 +473,72 @@ The terminology for the `ip_whitelist` and `ip_blacklist` annotations have been
+
+// For information about using the Secrets Store CSI Driver Operator to mount secrets from an external secrets store to a CSI volume, see xref:../nodes/index#nodes-pods-secrets-store[Providing sensitive data to pods by using an external secrets store].
+
+[id="ocp-4-18-release-notes-storage-pv-last-phase-transition-time_{context}"]
+==== Persistent volume last phase transition time parameter is generally available
+{product-title} 4.16 introduced a new parameter, `LastPhaseTransitionTime`, which has a timestamp that is updated every time a persistent volume (PV) transitions to a different phase (`pv.Status.Phase`). For {product-title} 4.18, this feature is generally available.
+
+For more information about using the persistent volume last phase transition time parameter, see xref:../storage/understanding-persistent-storage.adoc#persistent-volumes_understanding-persistent-storage[Persistent volumes].
+
+[id="ocp-4-18-release-notes-storage-volume-multi-vcenter-support_{context}"]
+==== Multiple vCenter support for vSphere CSI is generally available
+{product-title} 4.17 introduced the ability to deploy {product-title} across multiple vSphere clusters (vCenters) as a Technology Preview feature. In {product-title} 4.18, Multiple vCenter support is now generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-multi-vcenter-support-overview_persistent-storage-csi-vsphere[Multiple vCenter support for vSphere CSI] and xref:../installing/installing_vsphere/installation-config-parameters-vsphere.adoc[Installation configuration parameters for vSphere].
+
+[id="ocp-4-18-release-notes-storage-always-honor-reclaim-policy_{context}"]
+==== Always apply persistent volume reclaim policy (Technical Preview)
+Prior to {product-title} 4.18, the persistent volume (PV) reclaim policy was not always honored.
+
+For a bound PV-persistent volume claim (PVC) pair, the ordering of PV-PVC deletion determined whether the PV delete reclaim policy was applied or not. The PV applied the reclaim policy if the PVC was deleted prior to deleting the PV. However, if the PV was deleted prior to deleting the PVC, then the reclaim policy was not applied. As a result of that behavior, the associated storage asset in the external infrastructure was not removed.
+
+With {product-title} 4.18, the PV reclaim policy is consistently always applied. This feature has Technical Preview status.
```

Comment:
Should we have this feature listed under "Storage TechPreview features" table? cc. @duanwei33

Reply:
@radeore It's technically a feature, but it seems more to be addressing something that should have behaved this way all along, so I didn't list it. @gcharot @jsafrane Do you want me to list this in the features table?

Reply:
Leaving out of table.